test/test_mbxml_search.py

```diff
@@ -68,6 +68,11 @@ def testFields(self):
         one = res["release-list"][0]
         self.assertEqual("100", one["ext:score"])
 
+        # search results have a medium-list/track-count element
+        self.assertEqual("4", one["medium-track-count"])
```

Comment:
I see that we cast the `-count` elements to integers. Should we do the same here?